composer.json

```diff
@@ -17,26 +17,27 @@
         }
     ],
     "require": {
-        "php": ">=7.4",
-        "symfony/config": "^5.4 || ^6.0",
-        "symfony/dependency-injection": "^5.4 || ^6.0",
-        "symfony/form": "^5.4 || ^6.0",
-        "symfony/http-foundation": "^5.4 || ^6.0",
-        "symfony/http-kernel": "^5.4 || ^6.0",
-        "symfony/validator": "^5.4 || ^6.0",
+        "php": ">=8.0",
+        "symfony/config": "^5.4 || ^6.2@dev",
+        "symfony/dependency-injection": "^5.4 || ^6.2@dev",
+        "symfony/form": "^5.4 || ^6.2@dev",
+        "symfony/http-foundation": "^5.4 || ^6.2@dev",
+        "symfony/http-kernel": "^5.4 || ^6.2@dev",
+        "symfony/validator": "^5.4 || ^6.2@dev",
```

Comment:
Not so sure that introducing a `x@dev` dependency is a good idea. Why the `^6.2` wouldn't be enough ?

Reply:
To be compatible with next beta Symfony version.